packages/integration-tests/src/test-e2e/create_amplify.test.ts

```diff
@@ -6,175 +6,206 @@ import * as os from 'os';
 import { after, afterEach, before, beforeEach, describe, it } from 'node:test';
 import assert from 'assert';
 import { glob } from 'glob';
-
-void describe('create-amplify script', () => {
-  before(async () => {
-    // start a local npm proxy and publish the current codebase to the proxy
-    await execa('npm', ['run', 'clean:npm-proxy'], { stdio: 'inherit' });
-    await execa('npm', ['run', 'vend'], { stdio: 'inherit' });
-
-    // nuke the npx cache to ensure we are installing packages from the npm proxy
-    const { stdout } = await execa('npm', ['config', 'get', 'cache']);
-    const npxCacheLocation = path.join(stdout.toString().trim(), '_npx');
-
-    if (existsSync(npxCacheLocation)) {
-      await fs.rm(npxCacheLocation, { recursive: true });
-    }
-  });
-
-  after(async () => {
-    // stop the npm proxy
-    await execa('npm', ['run', 'stop:npm-proxy'], { stdio: 'inherit' });
-  });
-
-  let tempDir: string;
-  beforeEach(async () => {
-    tempDir = await fs.mkdtemp(path.join(os.tmpdir(), 'test-create-amplify'));
-  });
-
-  afterEach(async () => {
-    await fs.rm(tempDir, { recursive: true });
-  });
-
-  const initialStates = ['empty', 'module', 'commonjs'] as const;
-
-  initialStates.forEach((initialState) => {
-    void it(`installs expected packages and scaffolds expected files starting from ${initialState} project`, async () => {
-      if (initialState != 'empty') {
-        await fs.writeFile(
-          path.join(tempDir, 'package.json'),
-          JSON.stringify(
-            {
-              name: 'test_name',
-              version: '0.0.1',
-              type: initialState,
-            },
-            null,
-            2
-          )
-        );
+import { testConcurrencyLevel } from './test_concurrency.js';
+
+void describe(
+  'create-amplify script',
+  { concurrency: testConcurrencyLevel },
```

Comment:
Here's where the parallel magic happens. ref: https://nodejs.org/api/test.html#testname-options-fn